src/Makefile.test_util.include

```diff
@@ -19,7 +19,7 @@ TEST_UTIL_H = \
   test/util/transaction_utils.h \
   test/util/txmempool.h \
   test/util/validation.h \
-  test/util/wallet.h
+  wallet/test/util.h
```

Comment:
In cd62eddbcfba652a238374d62f70181cb3ad9075 "refactor: unify test/util/wallet.h with wallet/test/util.h"

nit: ISTM these should be guarded by `ENABLE_WALLET`? They aren't used by non-wallet benchmarks or tests, so it doesn't break anything.

This would also remove the need to have `#ifdef ENABLE_WALLET` in those files too.

Reply:
hmm yeah, could also remove them from `BITCOIN_TEST_SUITE` as `TEST_UTIL_H` is already included there.